openfisca_dummy_country/dummy_reforms.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+
+
+class neutralization_rsa(Reform):
+    def apply(self):
+        self.neutralize_column('rsa')
```

Comment:
It's tempting, but I don't think we should put PRs on the critical path of others. So let's see who gets merged first ;)